src/python/turicreate/toolkits/_image_feature_extractor.py

```diff
@@ -162,6 +157,10 @@ def consume_response(tf_out):
         request_queue = _Queue()
         response_queue = _Queue()
         def tf_worker():
+            from tensorflow import keras
+            model_path = self.ptModel.get_model_path('tensorflow')
+            self.model = keras.models.load_model(model_path)
```

Comment:
Loading the model on this thread seems to resolve an error that I was able to repro on Linux when running the IC tests after the DC tests....

Reply:
I think that's likely a symptom of one of the issue I'm looking into right now....

Reply:
As long as you think this change is not logically/structurally incorrect, doing it this way should also help us parallelize the keras model loading with data prep for the first batch

Reply:
Yes, I think this change is great. 

I think the issue you describe is the result of mismatch enabled/disabled V2 behavior.

Reply:
Alas, even with that change, Travis revealed a strange crash. Maybe that will somehow go away once we clear away all this enable/disable V2 stuff and the `reset_default_graph` question, but for now I suggest we back off from the multiple threads and just do the essential batching, to improve our memory footprint

Reply:
I'll check the perf on that.